blocks/editable/index.js

```diff
@@ -143,6 +155,13 @@ export default class Editable extends Component {
 
 	onInit() {
 		this.updateFocus();
+		this.registerCustomFormatters();
```

Comment:
What happens if the `formatters` prop change, should we resync the custom formatters in TinyMCE? If it's not possible, maybe we should just guard against updating this prop in `componentWillReceiveProps`?

Reply:
I've changed the prop from formatters to initialFormatters and added a console.error into componentWillReceiveProps, I forsee passing functions through this in future so it will be hard to determine if props have changed...